bin/diagnostics.bash

```diff
@@ -400,8 +420,18 @@ run_diagnostics_query_metric() {
     curl "${endpoint}/query" -k --header "${header}" --data-urlencode query="${1}" | jq
 }
 
-log_info "Running diagnostics..."
+if [[ -z "${CK8S_PGP_FP:-}" ]]; then
+    import_gpg_file "${gpg_file}"
+fi
+log_info "Using the following fingerprints:"
+log_info "${CK8S_PGP_FP}"
+
+file="${CK8S_CONFIG_PATH}/diagnostics-${cluster}-$(date +%y%m%d%H%M%S).log"
+touch "${file}"
+
 config_load "${cluster}"
+log_info "Running diagnostics..."
+export CK8S_AUTO_APPROVE=true
```

Comment:
Is there a reason why this is hardcoded instead of up to the user to set?

Reply:
Yes, otherwise the validation will trigger a prompt that is not visible when running the script since we are piping all output to a file in CK8S_CONFIG_PATH. The output from the validation will still be added to the diagnostics file, but with this it won't be seen as the script gets "stuck".

Reply:
And since we already do a validation before running diagnostics, the user will already know if there is something deviating in their config, and hopefully fix it or at least be aware of it.

Reply:
Makes sense. It's important to note that `CK8S_AUTO_APPROVE` doesn't just automatically approve validation failure prompts but _all_ prompts. So be careful with what you run as part of the diagnostics collection. :smile:

Reply:
Good point, I believe currently it is only the validation that has a prompt, but good to keep in mind. 
We should probably improve it somehow in the future, like better indication that the script is actually running and doing stuff, as it can take quite a while to run and sometimes it is unclear if it is running or stuck.